ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/ValueValidator.scala

```diff
@@ -47,9 +47,10 @@ object ValueValidator {
     case Sum.ContractId(cId) =>
       ContractId
         .fromString(cId)
-        .bimap(invalidArgument, Lf.ValueContractId(_))
+        .bimap(invalidArgument(definiteAnswer = Some(false)), Lf.ValueContractId(_))
```

Comment:
Just for my knowledge, why is this on `false`? Any documentation I would need to read? 
As in my mind this represents a malformed contract id so you can resubmit until the end of ages.

Reply:
The semantics of definite answers is explained in https://github.com/digital-asset/daml/pull/9567/files#diff-ed4ed30e4f593aaf9d013b39ed5272dfc0df08c548cc78f110fd4471eaa6f59cR143-R151. It applies even if the application had previously sent or will later send a different command for the same change ID.